### PR TITLE
Clarifying list continuation in ordered lists

### PR DESCRIPTION
**Pull request by MarciaRieferJohnston**

### CODE REVIEW

**Comment by MarciaRieferJohnston** [comment]
Hi, Dan (@mojavelinux). Thanks for requesting the review from Sarah (@graphitefriction). Thanks for creating such useful documentation. Our team has been using Asciidocs for the last year. Your descriptions and how-tos and examples have been helpful.

**Comment by MarciaRieferJohnston** [comment]
@mojavelinux Hi, Dan. Can you suggest others who might also be appropriate reviewers for this PR?

File: docs/modules/lists/pages/continuation.adoc [hunk 1]
```diff
@@ -16,17 +16,23 @@ include::example$complex.adoc[tag=indent]
 include::example$complex.adoc[tag=indent]
 ====
 
-TIP: When items contain more than one line of text, leave a blank line before the next item to make the list easier to read.
+TIP: When list items contain more than one line of text, leave a blank line between items to make the list easier to read while working in the code.
+
+Blank lines between items in an ordered list do not reset numbering to 1 as long as each adjacent line starts with a period. 
+Blank lines do reset numbering to 1 when something, such as a figure or a supporting paragraph, comes between two lines that start with a period.
+To keep numbering continuous in those cases—for example, if you're documenting steps in a procedure—put a `{plus}` symbol in the otherwise blank line. A `{plus}` used in this way is called a list continuation. For details on continuous numbering in an ordered list, see the https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/[Ordered Lists^] section of this documentation.
```
Comment (graphitefriction) [inline docs/modules/lists/pages/continuation.adoc:23]:
```suggestion
To keep numbering continuous in those cases—for example, if you're documenting steps in a procedure—put a `{plus}` symbol in the otherwise blank line. A `{plus}` used in this way is called a list continuation. For details on continuous numbering in an ordered list, see xref:ordered.adoc[].
```

Hi @MarciaRieferJohnston First off, thanks for helping improve our docs! I appreciate the time you're taking from other things I'm sure you could be doing.

In the suggestion above, I've replaced the link (`https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/[Ordered Lists^]`) with an xref macro. When linking between pages in docs.asciidoctor.org (no matter what project the page belongs to) we use the xref macro and Antora page ID structure for the target (`xref:page-id.adoc[]`). 

A page ID is "a unique and reliable set of identifying coordinates, organized in a standard sequence, that are constructed from the properties assigned to each page’s source file". :grimacing: I hope I didn't just send you running into the night. If you're still with me :crossed_fingers:, you can learn about page IDs here: https://docs.antora.org/antora/3.0/page/page-id/ and see more examples here: https://docs.antora.org/antora/3.0/page/module-and-page-xrefs/ if you're interested.

Long story short, all the pages you've edited in this MR are in the same [module](https://docs.antora.org/antora/3.0/module-directories/) (lists) so to create links between them, just specify the page's file name with .adoc in the xref target. For example, `xref:ordered.adoc[]` to link to the Ordered Lists page, or `xref:unordered.adoc[]` to link to the Unordered Lists page.

I also removed the link text in the suggestion. Antora will fill in the title of the page being referenced automatically when the site is generated.

File: docs/modules/lists/pages/ordered.adoc [hunk 2]
```diff
@@ -32,14 +32,24 @@ For instance, you can type:
...
 ----
 
-To present the items in reverse order, add the `reversed` option:
+If your ordered list includes figures or independent chunks of text, as is often the case when documenting steps in a procedure, you may find your steps resetting to 1 when you don't want them to. 
+This may happen, for example, after figures or multiparagraph steps. 
+In these cases, don't use the `start` attribute to continue the numbering.
+You don't want to have to manually update `start` numbers—or risk having the `start` numbers become wrong—every time you add or remove a step.
+Instead, use a list continuation—a `{plus}` symbol on its own line—between the numbered list items and all other content elements in your list.
+
+* For how-to details, see the https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/[List Continuation^] section of this documentation.
```
Comment (graphitefriction) [inline docs/modules/lists/pages/ordered.adoc:48]:
```suggestion
* For how-to details, see the xref:continuation.adoc[] section of this documentation.
```

File: docs/modules/lists/pages/continuation.adoc [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+This section applies to complex lists—ordered or unordered lists that have multiple lines of text per item or that contain a mix of content types—that you want to keep continuous. 
+
 == Indented lines
 
-Aside from nested lists, all of the list items you've seen only have one line of text.
-But like with regular paragraph text, the text in a list item can wrap across any number of lines, as long as all the lines are adjacent.
-The wrapped lines can be indented, and they will still be treated as normal paragraph text.
-For example:
+As with regular paragraph text, the text in a list item can wrap across any number of lines as long as all the lines are adjacent. 
+Multiple lines are combined into a paragraph and wrap as normal paragraph text even if they're indented, as shown in the third bullet in this example:
```
Comment (graphitefriction) [inline docs/modules/lists/pages/continuation.adoc:8]: This explanation is much more concise :tada: Thanks!

**Review by graphitefriction — CHANGES_REQUESTED** [review]

File: docs/modules/lists/pages/continuation.adoc (hunk 3 above)
Comment (MarciaRieferJohnston) [reply docs/modules/lists/pages/continuation.adoc:8]: You're welcome, Sarah. I'm glad that the new phrasing works for you. I hope that others find it helpful too.

**Comment by MarciaRieferJohnston** [comment]
@graphitefriction Thanks for fixing the cross-references, Sarah. I've committed your inline suggestions. Have I missed anything that  needs to be done before you approve this PR?

**Review by graphitefriction — APPROVED** [review]

**Comment by graphitefriction** [comment]
@MarciaRieferJohnston I don't think you've missed anything so I'm going to merge your PR. Thank you, again, for updating these docs!

**Comment by MarciaRieferJohnston** [comment]
@graphitefriction Thanks, Sarah!
